Makefile

```
@@ -5,6 +5,7 @@ DATE 			 ?= $(shell date +'%Y%m%d')
 IMAGE_TAG		 ?= $(RELEASE)_$(DATE)
 KUBECTL_BIN      ?= bin/kubectl
 KUBECTL_VERSION  ?= v1.23.11
+NOTEBOOK_REPO_BRANCH_BASE="https://raw.githubusercontent.com/opendatahub-io/notebooks/main"
```

```suggestion
NOTEBOOK_REPO_BRANCH_BASE ?= "https://raw.githubusercontent.com/opendatahub-io/notebooks/main"
```

lets make it generic, so other can run the test from their fork , if they want.

---

Oh, yeah, good catch! I wrote it the `bash` way :upside_down_face: . But it worked as expected too :smile: 

Anyway, I'll update. Thank you :slightly_smiling_face: